### ISSUE TITLE
Fixed height value (e.g. 100%) within groups works only when it's smaller than the group

### ISSUE THREAD

**Issue by sebastian-tapwork** [matt-curtis/Fluid-for-Sketch#129]
Hi, I'm having this issue since I'm using Fluid (thats a ling time ;) I want to give a rectangle which works as a background or container a height of 100% of the group. When the group becomes larger it works. The rectangle adapts to the increasing size of the group. When the group becomes smaller it doesn't do anything. But when I decrease the rectangle to a smaller size than the group it works. See attached gif. A fix of that would be so nice! Thank you
![aug -18-2017 11-15-56](https://user-images.githubusercontent.com/7447662/29452448-bb9e34b2-8406-11e7-848c-98e9c87635eb.gif)

**Comment by matt-curtis** [comment]
Yikes. Do you have a sample document?

**Comment by sebastian-tapwork** [comment]
yes. I attached it. the sample is behaving even more weird. something is definitely wrong with the height...
[fluid height bug example.zip](https://github.com/matt-curtis/Fluid-for-Sketch/files/1275065/fluid.height.bug.example.zip)
